docs/src/advanced-concepts/auxiliary-outputs.rst

```diff
@@ -0,0 +1,24 @@
+Auxiliary outputs
+=================
+
+These outputs, which are idenfified by the ``mts-models::aux::`` prefix,
+represent additional information that the model may provide. They are not
+conventional trainable outputs, and they often correspond to internal
+information that the model is capable of providing, such as its internal
+representation.
+
+The following auxiliary outputs that are currently supported
+by one or more architectures in the library:
+
+- ``mts-models::aux::last_layer_features``: The internal representation
+   of the model at the last layer, before the final linear transformation.
```

Comment:
this also needs a more precise definition of what's stored in this output. You can use https://docs.metatensor.org/latest/atomistic/outputs.html#energy for inspiration, but all the metadata should be specified here.

Reply:
Done